zypper_test.go

```diff
@@ -43,8 +46,8 @@ func TestIsZypperExitCodeSevere(t *testing.T) {
 		zypperExitNoRepos,
 		zypperExitZyppLocked,
 		zypperExitErrCommit,
-		zypperExitIndCapNotFound,
```

Comment:
@mssola : This has been classified as a severe exit code before. Keeping in mind what zypper's manpage says: What do you think?